index.js

```diff
@@ -0,0 +1,98 @@
+let clickedCard = null;
+let preventClick = false;
+let combosFound = 0;
+let clickcount = 0;
+let finalTime = 0;
+
+//TIMER CODE
+let startTime = Date.now();
+let second = 1000;
+let minute = second * 60;
+let hour = minute * 60;
+let timerCount = document.getElementById('timer');
```

Comment:
si esto no va a cambiar debería ser `const`

Reply:
let variables changed for const